### DISCUSSION TITLE
[FeatureRequest] Prevent "esc" key from exiting current drawing

### DISCUSSION THREAD

**Discussion by daeavelwyn** [pubpub-zz/ppInk#278] [Q&A]
Hello,

First of all, thanks for all the work you put into ppink, it's a so nice tool ! 

So, my problem is I often accidentaly hit the "esc" key while I'm drawing and it automatically exit ppink actual drawing. So, I'd like to be able to prevent "esc" from escaping the current drawing or have the ability to assign another shortcut to such function (shift+esc for example).

I hope you'll find the time to take this request in consideration,

best regards

**Comment by pubpub-zz** [comment]
you can change the hotkey in the options. Do not forget to save the setup when you will be happy. and do not hesitate to star the project

**Comment by daeavelwyn** [comment]
I tried changing the "close (in addition to alt+F4)" shortcut to shift+escape, but it doesn't prevent the single "esc" key to exit the current drawing. Maybe I missed something regarding shortcuts, could you indicate me the way I could disable the single "esc" key to exit the current drawing ?
